src/python/pants/option/global_options.py

```diff
@@ -724,7 +724,9 @@ class BootstrapOptions:
         ),
     )
     log_levels_by_target = DictOption[str](
-        daemon=True,
+        # TODO: While we would like this option to be fingerprinted for the daemon, the Rust side
+        # option parser does not support dict options. See #19832.
```

Comment:
What's the consequence of this? Someone might edit these log levels and they're not reflected if there's an already-running daemon? Based on this being a TODO, I'm imagining you think this is acceptable in the near term (I agree, if so)?

Reply:
Yea, I think that this is acceptable in the medium term (until #19832 can be resolved). If this were a more widely used option I'd probably feel otherwise.